src/components/Gesture.svelte

```diff
@@ -235,67 +234,81 @@
   onClose={cancelRecording}>
   <div class="flex flex-col space-y-10">
     <div class="space-y-10 w-70">
-      {#if showCountdown}
+      <div class={showCountdown ? 'visible' : 'invisible'}>
```

Comment:
Interesting change, I haven't thought too hard about how visibility is toggled. Out of interest, what is the advantage of changing the visibility using a `div` instead of us an `if`?

Reply:
This is kind of hacky at the moment until we come back to this dialog. It keeps the element in the DOM but invisible visually and to screen readers so there isn't any layout shift. The cancel button now at least stays in the same place during the countdown and recording.